src/vs/workbench/parts/terminal/common/terminalService.ts

```diff
@@ -276,8 +276,10 @@ export abstract class TerminalService implements ITerminalService {
 			this._onInstancesChanged.fire();
 
 			this._terminalTabs.forEach((t, i) => t.setVisible(i === this._activeTabIndex));
+			return instance;
 		} else {
```

Comment:
Since we're now returning, could you invert this if so it's structured like this:

```
if (!instance) {
  toast
  return
}

...
```

Reply:
Fixed in https://github.com/Microsoft/vscode/pull/65973/commits/56370116207131665632f8c74ddfc49efbe22fa9.